applications/serial_lte_modem/prj.conf

```diff
@@ -108,6 +108,9 @@ CONFIG_MQTT_CLEAN_SESSION=y
 CONFIG_AT_CMD_PARSER=y
 CONFIG_AT_MONITOR=y
 
+# Disable TF-M logging. TF-M logs may block when HWFC is used on UART1.
```

Comment:
Actually, block what? (in the comment)

Reply:
The log sending through uart uses blocking send, so nothing proceeds until send is complete.

Might be better like this:
```suggestion
# Disable TF-M logging. TF-M log sending will block until remote end is ready, when HWFC is used on UART1.
```

Reply:
Agreed.